src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -558,4 +558,9 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.removeAnswerRow('uniqueKey', true, 'url', 'name', 'code', true);
     expect(spy).toHaveBeenCalled();
   });
+  it('should call the removeAnswerRow function and remove the anwser row when a user clicks'
+  + 'the remove button while creating a Q-A concept', () => {
+    wrapper.find('button#removeAnswer').simulate('click');
+    expect(props.removeAnswer).toHaveBeenCalledWith(props.selectedAnswers[0].frontEndUniqueKey);
```

Comment:
What should have been called? removeAnswer or removeAnswerRow?

Reply:
removeAnswer should be called

Reply:
Is that what your description says?

Reply:
Yes that is what it currently says, I'm sorry, I'm thinking I should have adjusted the description before commenting to avoid the unnecessary back and forth